hw/ip/otbn/doc/_index.md

```diff
@@ -417,6 +417,21 @@ Additionally, instructions and data stored in the instruction and data memory, r
 
 Refer to the [Data Integrity Protection]({{<relref "#design-details-data-integrity-protection">}}) section for details of how the data integrity protections are implemented.
 
+## Secure Wipe
+
+OTBN provides a mechanism to securely wipe all state it stores, including the instruction memory.
+
+This mechanism is a superset of other secure wipe mechanisms:
```

Comment:
I'm not sure this is quite the right phrasing. Maybe just "This wiping is split into three parts"?